frontend/src/components/Common/TabMenu/TabMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button, Text } from '@fun-eat/design-system';
+import { useState } from 'react';
+import { css, styled } from 'styled-components';
+
+interface TabMenuProps {
+  menuVariant: 'food' | 'store';
+  menuList: string[];
+}
+
+const TabMenu = ({ menuList, menuVariant }: TabMenuProps) => {
+  const [selectedMenu, setSelectedMenu] = useState(0);
+
+  const selectMenu = (index: number) => {
+    setSelectedMenu(index);
+  };
+
+  return (
+    <TabMenuContainer>
+      {menuList.map((menu, index) => (
+        <TabMenuButton
+          key={menu}
```

Comment:
```suggestion
        <TabMenuButton
          key={menu}
          type="button"
```

button 태그의 타입을 주면 더 좋을거 같아요 😊